packages/sfui/frameworks/react/components/VsfListItemFilterSwitch/VsfListItemFilterSwitch.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import { VsfListItemMenu } from '../VsfListItemMenu';
+import type { VsfListItemFilterSwitchProps } from './types';
+import { VsfListItemFilterSwitchVariant } from './types';
+import { VsfSwitch } from '../VsfSwitch';
+
+export default function VsfListItemFilterSwitch({
+  label,
+  counter,
+  size,
+  variant = VsfListItemFilterSwitchVariant.prefix,
+  secondaryText,
+  disabled,
+  className,
+  truncate,
+  value,
+  name,
+  children,
+  checked,
+  onChange = () => '',
+  ...attributes
+}: VsfListItemFilterSwitchProps): JSX.Element {
+  const [checkedState, setCheckedState] = useState(false);
```

Comment:
I feel like we missing default value here. Also, imo a better name of the state would be "selected" to reflect ListItemMenu props.
```suggestion
  const [selected, setSelected] = useState(defaultValue);
```

Reply:
ok, I'll chnge it